pkg/kubectl/cmd/exec.go

```diff
@@ -32,17 +32,20 @@ import (
 )
 
 const (
-	exec_example = `// get output from running 'date' in ruby-container from pod 123456-7890
-$ kubectl exec -p 123456-7890 -c ruby-container date
+	exec_example = `// get output from running 'date' from pod 123456-7890, using the first container by default
+$ kubectl exec 123456-7890 date
+	
+// get output from running 'date' in ruby-container from pod 123456-7890
+$ kubectl exec 123456-7890 -c ruby-container date
```

Comment:
I mentioned this in #8448:

I'd prefer to drop the ```-c``` but the cobra parsing makes it impossible to tell the difference between:

```kubectl exec <pod> <container> -- <cmd>```
and
```kubectl exec <pod> --- <cmd> <arg>```

So I think we have to keep the ```-c``` for now.